_posts/2017-06-05-rice.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: petition-page
+title: "Tell Kellogg 'People need their rice'"
```

Comment:
Our style is usually a colon instead of quotation marks, e.g., "Tell Kellogg: People Need Their Rice"